### ISSUE TITLE
ON_CONFLICT UPDATE SET confuses parser

### ISSUE THREAD

**Issue by HyperGH** [macbre/sql-metadata#197]
When using PostgreSQL:
```py
>>> sql = 'INSERT INTO global_config (entry_id) VALUES ($1) ON CONFLICT (entry_id) UPDATE SET entry_id = $1'
>>> parser = Parser(sql)
>>> parser.tables
['global_config', 'SET']
```
It seems to think that `SET` is also a tablename, but in Postgres you can omit the tablename/`WHERE` in an `ON CONFLICT` clause as it only executes the code if it conflicts.

**Comment by collerek** [comment]
Yes, as of now we don't support `on conflict` clause - hard to tell if that's a bug or enhancement but let's go with a bug, and we will try to add this feature.

**Comment by HyperGH** [comment]
I also noticed that `DELETE FROM xy WHERE` statements also raise an error, not sure if I should open an issue for that?
